sapl/test_config.py

```diff
@@ -0,0 +1,25 @@
+from .settings import EMAIL_PORT
+from .settings import SECRET_KEY
+from .settings import DEBUG
+from .settings import DATABASES
+from .settings import EMAIL_USE_TLS
+from .settings import EMAIL_HOST
+from .settings import EMAIL_HOST_USER
+from .settings import EMAIL_HOST_PASSWORD
+
+data = DATABASES.get('default')
+
+
+def test_config():
```

Comment:
@eduardocalil Agora eu bugei: como este teste funcionou? Vc criou um .env na tua máquina???? Se for isso tem que ver uma forma de criar um teste que não dependa de um arquivo .env local, isto é, que vc possa informar um arquivo do tipo "teste.env" pra rotina de teste.